ui/scripts/vpc.js

```diff
@@ -2652,7 +2652,8 @@
                                                     url: createURL('listStaticRoutes'),
                                                     data: {
                                                         gatewayid: args.context.vpcGateways[0].id,
-                                                        listAll: true
+                                                        listAll: true,
+                                                        state: "Active"
```

Comment:
@Spaceman1984 can you send the related change to Primate as well cc @davidjumani